### PR TITLE
RI-3530 regression tests refactoring

### PR DESCRIPTION
**Pull request by vlad-dargel**
Refactoring of e2e regression tests from https://redislabs.atlassian.net/browse/RI-3530
Fixed:
1) Many small tests are merged to save test execution time
2) Comments are reduced to one formatting
3) chance.word() replaced by common.generateWord()
4) Error messages of expects are fixed (previously most of them were incorrect)
5) "rte: rte.standalone" moved from tests meta to fixture meta
6) Different updates for some tests

### CODE REVIEW

File: tests/e2e/tests/regression/cli/cli-promote-workbench.ts [hunk 1]
```diff
@@ -21,16 +21,16 @@ fixture `Promote workbench in CLI`
...
-    // Open CLI
-    await t.click(cliPage.cliExpandButton);
-    await t.expect(cliPage.workbenchLink.parent().textContent).eql('Try Workbench, our advanced CLI. Check out our Quick Guides to learn more about Redis capabilities.');
-    // Verify that user is redirected to Workbench page clicking on workbench link in CLI
-    await t.click(cliPage.workbenchLink);
-    await t.expect(workbenchPage.expandArea.exists).ok('Workbench page is opened');
-    // Verify that CLI panel is minimized after redirection to workbench from CLI
-    await t.expect(cliPage.cliPanel.visible).notOk('Closed CLI');
-});
+// test('Verify that users can see workbench promotion message when they open CLI', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/cli/cli-promote-workbench.ts:24]: ?

File: tests/e2e/tests/regression/workbench/autocomplete.e2e.ts [hunk 2]
```diff
@@ -8,84 +8,87 @@ const myRedisDatabasePage = new MyRedisDatabasePage();
...
+
+    await t.typeText(workbenchPage.queryInput, command2, { replace: true });
+    // Check that hint with arguments are displayed
+    await t.expect(workbenchPage.monacoHintWithArguments.visible).ok('Hints with arguments are not displayed');
+    // Remove hints with arguments
+    await t.pressKey('esc');
+    // Verify that user can close the static list of arguments by pressing “ESC”
+    await t.expect(workbenchPage.monacoHintWithArguments.visible).notOk('Hints with arguments are still displayed');
+});
+// test('Verify that user can close the static list of arguments by pressing “ESC”', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/regression/workbench/autocomplete.e2e.ts:65]: ?

**Review by ofersteinberg — CHANGES_REQUESTED** [review]

File: tests/e2e/tests/regression/cli/cli-promote-workbench.ts (hunk 1 above)
Comment (vlad-dargel) [reply tests/e2e/tests/regression/cli/cli-promote-workbench.ts]: thanks for noticing, I forgot to delete some already merged tests

File: tests/e2e/tests/regression/workbench/autocomplete.e2e.ts (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/tests/regression/workbench/autocomplete.e2e.ts]: thanks for noticing, I forgot to delete some already merged tests

**Review by ofersteinberg — APPROVED** [review]

File: tests/e2e/tests/regression/browser/database-overview-keys.e2e.ts [hunk 3]
```diff
@@ -56,21 +54,19 @@ fixture `Database overview`
...
-test
-    .meta({ rte: rte.standalone })('Verify that user can see total number of keys and not it current logical database (if there are no any keys in other logical DBs)', async t => {
-        //Open Database
+
+        // Open Database
         await t.click(myRedisDatabasePage.myRedisDBButton);
         await myRedisDatabasePage.clickOnDBByName(ossStandaloneConfig.databaseName);
         await t.hover(workbenchPage.overviewTotalKeys);
-        //Verify that user can see only total number of keys
+        // Verify that user can see total number of keys and not it current logical database (if there are no any keys in other logical DBs)
```
[the remark this answers is not in the document]
Comment (vlad-dargel) [reply tests/e2e/tests/regression/browser/database-overview-keys.e2e.ts:69]: Verify that user can see only total number of keys - it's not the test name in testplan, just comment)

File: tests/e2e/tests/regression/browser/format-switcher.e2e.ts [hunk 4]
```diff
@@ -68,16 +68,16 @@ test
         // Verify that formatters selection is saved when user switches between databases
         await t.expect(browserPage.formatSwitcher.withExactText('JSON').visible).ok('Formatter value is not saved');
     });
-test('Verify that user don`t see format switcher for JSON, GRAPH, TS keys', async t => {
+test('Verify that user can see switcher icon for narrow screen and tooltip by hovering', async t => {
```
[the remark this answers is not in the document]
Comment (vlad-dargel) [reply tests/e2e/tests/regression/browser/format-switcher.e2e.ts:71]: I moved test "Verify that user don`t see format switcher for JSON, GRAPH, TS keys" below, it is included into "Verify that user can see switcher icon for narrow screen and tooltip by hovering"

**Review by AlenaNaboka — APPROVED** [review]

**Pull request merged**